webapi/Controllers/ChatHistoryController.cs

```diff
@@ -69,18 +73,19 @@ public class ChatHistoryController : ControllerBase
     /// <param name="chatParameter">Contains the title of the chat.</param>
     /// <returns>The HTTP action result.</returns>
     [HttpPost]
-    [Route("chatSession/create")]
+    [Route("chatSessions")]
```

Comment:
Revert all route changes - will be done in a subsequent PR

Reply:
Why? I'm vibing the new routes

Reply:
Because it's a breaking change that customers need to be aware of. We need to provide them with a way to transition without downtime. We need to ensure that we make such changes only once (do it right) and thoughtfully. But we don't want to hold the rest of the security improvements until then.